### PR TITLE
Feature/itm 302

### PR DESCRIPTION
**Pull request by mrb-itminds**
depends on #50 

Added Service Library Page with ServiceCards placed in a responsive table

### CODE REVIEW

File: frontend/src/components/ServiceLibrary/ServiceLibrary.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          </HStack>
+          <Flex width="full">
+            <Box p="4">Services</Box>
+            <Spacer />
+            <Box onClick={() => setAllShown(!allShown)} p="4">
+              {!allShown ? (
+                <Box cursor="pointer">View all ({services.length})</Box>
+              ) : (
+                <Box cursor="pointer">Hide all ({services.length})</Box>
+              )}
```
Comment (objmj-itminds) [inline frontend/src/components/ServiceLibrary/ServiceLibrary.tsx:39]: Scope the if a bit more like so
```
 <Box cursor="pointer">
   {!allShown ? "View all" : "Hide all"} ({services.length})
 </Box>
```

File: frontend/src/components/ServiceLibrary/ServiceLibrary.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+              </SimpleGrid>
+            ) : (
+              <SimpleGrid width="full" minChildWidth="300px" spacingX="20px" spacingY="50px">
+                {services.slice(0, 5).map((Service: ServiceIdDto) => (
+                  <Center key={Service.id}>
+                    <LibraryCard service={Service}></LibraryCard>
+                  </Center>
+                ))}
+              </SimpleGrid>
+            )}
```
Comment (objmj-itminds) [inline frontend/src/components/ServiceLibrary/ServiceLibrary.tsx:59]: Same here. I can only see the `.slice` being the difference, so scope the if down to that part
```
{(allShown ? services : services.slice(0,5)).map ... }
```

**Review by objmj-itminds — CHANGES_REQUESTED** [review]

File: frontend/src/components/ServiceLibrary/ServiceLibrary.tsx (hunk 1 above)
Comment (mrb-itminds) [reply frontend/src/components/ServiceLibrary/ServiceLibrary.tsx]: Makes sense my bad ^^

File: frontend/src/components/ServiceLibrary/ServiceLibrary.tsx (hunk 2 above)
Comment (mrb-itminds) [reply frontend/src/components/ServiceLibrary/ServiceLibrary.tsx]: Makes sense my bad ^^

**Review by objmj-itminds — APPROVED** [review]

**Pull request merged**